### PR TITLE
Transformation templates docs

### PR DESCRIPTION
**Pull request by svix-lucho**
In the 'Transformations' page, add section about Transformation Templates

Screenshot 2023-08-08 at 11 25 01 AM
Screenshot 2023-08-08 at 11 25 11 AM

### CODE REVIEW

File: docs/transformations.mdx [hunk 1]
```diff
@@ -48,8 +48,65 @@ function handler(webhook) {
...
+
+To create a template, you need to provide the following details:
+
+- `name`, `description` and `logo url` - Describe the integration to your customers.
+- `Instructions` and `Learn more link` - Explain how to use and customize the template. Your customers may need to set a specific endpoint when using the template. This is a good place to let them know how to get that endpoint.
+- `Event Types` - Specify which events are going to be supported out-of-the-box by the integration. 
+- `Transformation Code` - This is the glue code that will turn incoming webhook events into useful payloads for your integration. For example, if you are building a transformation template that sends Slack messages every time a new invoice is created, the transformation 
+will read `invoice.created` events and return a payload that works for Slack, like:
+```js
+{ "text": `An invoice of $${webhook.payload.amount} has been created.` }
```
Comment (svix-gabriel) [inline docs/transformations.mdx:75]: Is it actually two $ or is that typo?

File: docs/transformations.mdx [hunk 2]
```diff
@@ -48,8 +48,65 @@ function handler(webhook) {
...
+    webhook.payload = {
+      text: `${webhook.payload.name} deleted an invoice`
+    }    
+  }
+
+  return webhook
+}
+```
+
+Your customers will be able to customize the transformation code and the event types they listen to. Your definition should act as a (working!) starting point they can use.
```
Comment (svix-gabriel) [inline docs/transformations.mdx:106]:
```suggestion
Your customers will be able to customize the transformation code and the event types they listen to. Your definition should act as a working starting point they can use. 
```

File: docs/transformations.mdx [hunk 3]
```diff
@@ -48,8 +48,65 @@ function handler(webhook) {
 	if (webhook.payload.customUrl) {
 		webhook.url = webhook.payload.customUrl;
 	}
-	return webhook;
+	return webhook;
```
Comment (tasn) [inline docs/transformations.mdx:51]:
```suggestion
	return webhook;
```

**Review by svix-gabriel — APPROVED** [review]

File: docs/img/transformation-templates-list.png
Comment (tasn) [inline docs/img/transformation-templates-list.png:1]: Remove the sidebar from the screenshot or fix it as discussed. It currently contains the weird naming.

Comment (svix-lucho) [reply docs/img/transformation-templates-list.png:1]: Coming up in https://github.com/svix/monorepo-private/pull/6056 - I'll update the screenshot here once I get that PR in

**Pull request merged**